gadgets/trace_exec/test/integration/trace_exec_test.go

```diff
@@ -117,13 +118,14 @@ int main(int argc, char *argv[], char **envp) {
 	buildCmd := fmt.Sprintf("echo %s | base64 -d > exec.c && gcc -Wall -o /bin/exec-syscall exec.c", progBase64)
 	sleep1Args := []string{"/usr/bin/sleep", "0.4"}
 	sleep2Args := []string{"/usr/bin/sleep", "0.6"}
+	scriptsCmd := "echo '#!/bin/sh\nls' > /bin/with_shabeng.sh ; /dev/script.sh echo ls > /dev/script.sh ; echo ls > /bin/script.sh ; chmod +x /dev/script.sh /bin/script.sh /bin/with_shabeng.sh ; /dev/script.sh ; /bin/script.sh ; /bin/with_shabeng.sh;"
```

Comment:
- s/shabeng/shebeng/g
- spurious `/dev/script.sh` . Or did I miss something?

```suggestion
	scriptsCmd := "echo '#!/bin/sh\nls' > /bin/with_shebeng.sh ; echo ls > /dev/script.sh ; echo ls > /bin/script.sh ; chmod +x /dev/script.sh /bin/script.sh /bin/with_shebeng.sh ; /dev/script.sh ; /bin/script.sh ; /bin/with_shebeng.sh;"
```